app/Models/ContestJudgeCategoryVote.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property ContestJudgeCategory $category
```

Comment:
For reference, stuff that goes through relations are readonly on the actual property itself and should be `@property-read`